doc/manual/src/package-management/terminology.md

```diff
@@ -0,0 +1,28 @@
+# Terminology
+
+A *local store* exists on the local filesystem of the machine where
+Nix is invoked.  The `/nix/store` directory is one example of a
+local store.  You can use other local stores by passing the
+`--store` flag to `nix`.
+
+A *remote store* is a store which exists anywhere other than the
+local filesystem.  One example is the `/nix/store` directory on
+another machine, accessed via `ssh` or served by the `nix-serve`
+Perl script.
+
+A *binary cache* is a remote store which is not the local store of
+any machine.  Examples of binary caches include S3 buckets and the
+[NixOS binary cache](https://cache.nixos.org).  Binary caches use a
+disk layout that is different from local stores; in particular, they
+keep metadata and signatures in `.narinfo` files rather than in
+`/nix/var/nix/db`.
+
+A *substituter* is a store other than `/nix/store` from which Nix will
+copy the realisation of a derivation instead of building it.  Nix will
+not copy a realisation from a remote store unless one of the following
+is true:
```

Comment:
> a store other than `/nix/store`
So a remote store? At least, as someone who has trouble to understand this part of Nix, I'm not sure what this refers to.

> the realisation of a derivation
I presume that this should be written as [store object](https://nixos.org/manual/nix/unstable/glossary.html#gloss-store-object).

Reply:
> So a remote store?

Or a local store:

```
mkdir ~/other-local-store
nix --store ~/other-local-store/ copy --from https://cache.nixos.org /nix/store/i46mrzinxi9a5incliwhksmk947ff4wn-stdenv-bootstrap-tools-mipsel-unknown-linux-gnu
nix copy --from ~/other-local-store/ /nix/store/i46mrzinxi9a5incliwhksmk947ff4wn-stdenv-bootstrap-tools-mipsel-unknown-linux-gnu
```



> > the realisation of a derivation
> 
> I presume that this should be written as [store object](https://nixos.org/manual/nix/unstable/glossary.html#gloss-store-object).

Yes!

Reply:
implemented: aae771c

Reply:
Thanks for the examples! I found "_a store other than `/nix/store` _" confusing, but not sure how to succinctly say "Nix store at a non-default location"...